### DISCUSSION TITLE
Who wants to be hired (April 2023)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#47868] [Show and tell]
# What is this?
This discussion is for Next.js developers looking for work.

# Candidates

Share your information if you are looking for work. Please use this format:

  Location:
  Remote:
  Willing to relocate:
  Technologies:
  Resume/CV:
  Website:
  Email:

Good luck with interviewing!

---

If you got hired through this job board, please reach out to me (hassan@vercel.com) so I can send you some swag :)

If you're a company looking to post a role, use #47869.

*If you're looking for last month's discussion, see #46890. We've had yet another [*success story*](https://twitter.com/nutlope/status/1610667757565083648) of a developer being hired!*

**Comment by jensen** [comment]
Location: Vancouver, BC
Remote: Yes
Willing to relocate: No
Technologies: Full-Stack JavaScript (Check GitHub)
Resume/CV: https://www.linkedin.com/in/karljensendev/
Website: https://github.com/jensen

**I am only looking for contract work at this time.**

**Comment by Vancey567** [comment]
Location: Assam, India
Remote: Yes
Willing to relocate: Yes
Technologies: HTML, CSS, JavaScript, Reactjs, MongoDB, Node.js, Express.js, AWS(familiar)
Resume: https://drive.google.com/file/d/1vOeO3RAfisbvGZwjgaenXT7pzferHTrx/view?usp=sharing
LinkedIn: https://www.linkedin.com/in/jay-mishra-4a15b01a9
Github: https://github.com/vancey567

I am only looking for an internship as well as full-time roles.
Please feel free to connect with me on LinkedIn or email me at jayisatwork1@gmail.com.

**Comment by edward32tnt** [comment]
I'm a Full Stack Engineer, Nodejs is my primary development language, I am curious about learning new things.

Location: Shanghai, China.
Remote: Yes
Willing to relocate: No

Technologies: 
Nextjs, React, Vue, TailwindCSS, Webpack, Vite
Node, Koa, Python, Django, GraphQL, MongoDb, MySQL, Redis, Memcache
Nginx, Linux, HyGraph, Vercel, CICD, Docker, kubernetes, Cloud Native

Resume/CV: https://portfolio.edward32tnt.me
Linkedin: https://www.linkedin.com/in/zhaochunyu
Website: https://portfolio.edward32tnt.me
Email: edward32tnt@126.com

Thank you, contact me for more info.

**Comment by CharlesOuverleaux** [comment]
Hi! I'm Charles, a skilled and results-driven software engineer with a background in business and marketing strategy spanning over 6 years. I have a passion for building responsive websites and applications. My focus on enhancing user experiences through effective problem-solving enables me to deliver optimal user solutions.

Location: Berlin
Remote: Onsite, Hybrid and remote
Willing to relocate: No 
Technologies: 
- HTML, CSS, Tailwind, bootstrap
- Javascript, TypeScript
- React, Next.js
- Jest, Storybook, Sanity
- Ruby, Ruby on Rails, SQL (mySQL, PostgreSQL)
- Git, Github

Resume/CV: [CharlesOuverleaux_CV.pdf](https://github.com/vercel/next.js/files/11148489/CharlesOuverleaux_CV.pdf)
Website: https://charlesouverleaux.co/
Email: charlesantoine.ouverleaux@gmail.com

Please feel free to connect with me on LinkedIn or email me

**Comment by sabinbaniya** [comment]
Hey there, I'm Sabin. A full-stack js developer with a primary focus on UI & front-end. I have been working mostly with Next.js for over a year now in terms of professional experience.

Location: Nepal
Remote: Yes
Willing to relocate: No
Technologies: Next.JS, React.JS, Node.JS, MongoDB, Tailwind CSS, Typescript, etc.
Resume/CV: 
Website: https://sabinbaniya.com.np
Email: baniya.sabinn@gmail.com

**Comment by abhishekDev047** [comment]
Location: India
Remote:Yes
Willing to relocate: Yes
Technologies: React.js, Next.sj, node.js, tailwindcss, javascript, bootstrap, css, yarn, git.
Resume/CV: i'll mail you further 
Website: mail 
Email: abhishekkumar.in047@gmail.com

**Comment by Cedar-81** [comment]
Location: Nigeria
Remote: Yes
Willing to relocate: Yes
Technologies: Nextjs, React, TailwindCSS, Astro, Nodejs, ASP.NET(learning)
Resume/CV: https://docs.google.com/document/d/17UiuvD80S9IHcs3nKf6OyHEMoRAw2clDq5LJ9be7c18/edit?usp=sharing
Website: https://cedar.readate.org/
Email: cedarwudnovels@gmail.com

I am looking to work in intern and junior frontend dev roles and I'm willing to work for free as an intern as I am mostly interested in gaining real-world experience

**Comment by avinashvagh** [comment]
Location: Mahuva, Gujarat, India
Remote: Yes
Willing to relocate: Yes
Technologies: Nextjs, Reactjs, TailwindCSS, Typescript, Git, GitHub, 
Resume/CV: https://drive.google.com/file/d/19yTs1Nt3edsOaWG3NzTasR9y5VOvTSSo/view?usp=share_link
Website: https://www.nextyron.com
Email: coefeewithav@gmail.com

**Comment by jessemoe** [comment]
Location: China
Remote: Yes
Willing to relocate: Yes
Technologies: Nextjs, Reactjs, TailwindCSS, Typescript, Git, Node.js,
Resume/CV: https://1drv.ms/b/s!AmOR_poc9NLXgQZLCX1iyvOtkaSa?e=yQDrW2
Website: https://github.com/stark-eth
Email: [songgao.cao@outlook.com](mailto:songgao.cao@outlook.com)

**Comment by hunterunger** [comment]
Location: Calgary, Canada
Remote: Both work
Willing to relocate: False
Technologies: Databases, APIs, cloud machines, local tunnels, frontend, mobile, desktop, AI, content writing, 3D/2D graphics, etc (more detailed tech stack breakdown on LinkedIn)
Resume/CV: https://www.linkedin.com/in/hunterunger/
Website: https://oxen.dev/
Email: (Contact me on my website or LinkedIn)

**Comment by Podfelix** [comment]
Error: Cannot find module 'next/dist/compiled/jest-worker'
Require stack:
- /app/node_modules/next/dist/server/lib/squoosh/main.js
- /app/node_modules/next/dist/server/image-optimizer.js
- /app/node_modules/next/dist/server/next-server.js
- /app/server.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1026:15)
    at Function.mod._resolveFilename (/app/node_modules/next/dist/build/webpack/require-hook.js:23:32)
    at Function.Module._load (node:internal/modules/cjs/loader:871:27)
    at Module.require (node:internal/modules/cjs/loader:1098:19)
    at require (node:internal/modules/cjs/helpers:108:18)
    at Object.<anonymous> (/app/node_modules/next/dist/server/lib/squoosh/main.js:8:19)
    at Module._compile (node:internal/modules/cjs/loader:1196:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1250:10)
    at Module.load (node:internal/modules/cjs/loader:1074:32)
    at Function.Module._load (node:internal/modules/cjs/loader:909:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/app/node_modules/next/dist/server/lib/squoosh/main.js',
    '/app/node_modules/next/dist/server/image-optimizer.js',
    '/app/node_modules/next/dist/server/next-server.js',
    '/app/server.js'
  ]
}

**Comment by sunil-9** [comment]
Location: Nepal
Remote: yes
Willing to relocate: yes
Technologies:  Nextjs, Typescript, Git, Node.js, DevOps, Linux 
Resume/CV: https://drive.google.com/file/d/1SBxpt3V-QqQg7PJLTHoUxcTVvFsJQbY_/view
Website: https://sapkotasunil.com.np/
Email: sunilsapkota9@gmail.com

**Comment by aaabbbcc1995** [comment]
Hi,I am a front-end engineer from China, and I am current looking for remote job as FE or I am willing to become a fullstack engineer in a short time.
In the past year, I am working in a web3 company about nfts.

- I develop a web3 nft wallet in the past one year from 0 to 1, and users ups to 1 million nowadays.
- And I build 10+ nfts selling H5 webs for our customers, some of them are top brands in the world.
- And I build a saas system from 0 to 1 to help our customers to build a nfts h5 selling web in a easy way.
- And I build a UI libray for our wallet web product, and I use verdaccio/pm2/dumi/typescript build a private npm repo.

And hope that we can have a opportunity to know more about each others.
Location: Shanghai, China
Remote: Yes
Willing to relocate: No
Technologies: HTML, CSS, JavasScript, TypeScript, Verdaccio, Pm2, Dumi, React
Resume/CV: https://www.linkedin.cn/incareer/in/zaiquan-wen-116b43170/
Email: [wenzaiquan1995@gmail.com](mailto:wenzaiquan1995@gmail.com)

**Comment by NikharPandya** [comment]
Location: Ontario, CA
Remote: Yes
Willing to relocate: no
Technologies: Html, Css, TailwindCSS, Javascript(ES6), React js(18 and prior), Next.js (13 and above), Github, Vercel, Wordpress, MongoDB, Mysql
Resume/CV: https://www.linkedin.com/in/nikhar-pandya/
Website: https://www.nikharpandya.com/
Email: pandyanikhar@gmail.com

**Comment by Mitsunee** [comment]
Location: Germany
Remote: preferred
Willing to relocate: no
Technologies: Linux (Ubuntu/Debian, CentOS/Fedora), JavaScript/TypeScript (10+ years of hobby experience), React/Preact, Next.js, Astro, basic knowledge of Redis
Resume/CV: Please contact
Website: https://www.mitsunee.com
Email: (see website above)

**Comment by matrv** [comment]
Location: Russia
Remote: Yes
Willing to relocate: Yes 
Technologies: Next.js, React, SvelteKit, Vue, Solid.js, Tailwind CSS, tRPC
Resume/CV: https://resume.ronanru.com/resume.pdf
Website: https://ronanru.com
Email: ronanru@duck.com

**Comment by CamBrown101** [comment]
Hello there!

I am an experienced developer based in Vancouver, BC, specializing in web development with a growing proficiency in iOS development. I am currently seeking new opportunities, whether full-time or contract-based, to contribute my skills and knowledge to innovative projects.

If you have any questions or are interested in discussing potential opportunities, please feel free to contact me via email. I would be more than happy to connect with you and explore how I can bring value to your team or project.

Thank you,

Location: Vancouver, BC
Remote: Yes
Willing to relocate: No
Technologies: Full-Stack JavaScript, TypeScript, Next.js, React, Vue, Swift, Tailwind, Ruby on Rails
Resume/CV: https://www.linkedin.com/in/cameron-brown-101/
Website: https://cambrown101.github.io/personal-website/
Email: cameron-brown@live.com

**Comment by waldothedeveloper** [comment]
Location: Miami, USA
Remote: Yes
Willing to relocate: No
Technologies: Full-Stack JavaScript, TailwindCSS, NextJS, Novu, Firebase, Github, Pusher.
Resume/CV: https://drive.google.com/file/d/12kR_B7Ty5tak7vYJAbVu5i7CY8Smwmq9/view?usp=sharing
Website: https://github.com/waldothedeveloper
Email: [waldothedeveloper@gmail.com](mailto:waldothedeveloper@gmail.com)

**Comment by jordan716** [comment]
Hello,
My name is Jordan Engle and I'm a mid-senior front-end engineer with 8 years of professional experience in front-end development.

Location: Weston, OH
Remote: Yes
Willing to relocate: No
Technologies: JavaScript, TypeScript, React, NextJS, Tailwind CSS, RESTful APIs, GraphQL, AWS, and other relevant skills.
Linkedin: https://www.linkedin.com/in/jordan-engle-front-end-developer
Email: [jordanengle716@gmail.com](mailto:jordanengle716@gmail.com)
Phone: 7185142772

**Comment by Xelaflash** [comment]
Hello,
I'm a Frontend Web Developer with 5 years of experience in the industry looking for work freelance or long term position.
While I've started my journey as a full-stack developer working with **Ruby on Rails,** I've now been focusing on **Frontend** development through **React,** the **JAMSTACK** and its associated framework like **Gatsby** or **NextJs** (my favorite).

Autonomous, proactive and problem-solving, I'll be happy to jump on a call and discuss your project or my profile with you.


Location:  NYC, NY, USA
Remote: remote | hybrid | on-site (nyc)
Willing to relocate: No
Technologies: Javascript, Typescript,  React, NextJS, Tailwind CSS, APIs, GraphQL, mongodb, redux... and ecosystem tooling.
Resume/CV: [Resume](https://alexgwebdev.notion.site/Alexandre-Gros-e06bc5d76e604dadaee7559f07500b30)
Website: [alexgwebdev.com](https://www.alexgwebdev.com/)
Linkedin: [Linkedin](https://www.linkedin.com/in/alexgwebdev/)
Email: [contact@alexgwebdev.com](mailto:contact@alexgwebdev.com)

**Comment by ibravoh149** [comment]
Location: Lagos, Nigeria
Remote: true
Willing to relocate: true
Technologies: html/css, Javascript, typescript, react native, react, nextjs, nodejs, nestjs,mongodb, tailwindCSS, redis,rabbitmq
Resume/CV: https://www.linkedin.com/in/ibrahim-iliyas-5a6a2b109
Website: https://github.com/ibravoh149
Email: [ibravoh149@gmail.com](mailto:ibravoh149@gmail.com)

**Comment by Apestein** [comment]
Location: Atlanta, GA
Remote: Remote/Hybrid only
Willing to relocate: No
Technologies: NextJS/Typescript/Tailwind, for rest of stack see website/github.
Resume/CV: https://www.linkedin.com/in/luc-ledo/
Website: https://lucledo.dev
Email: lucledo@protonmail.com

**Summary**
Full-stack web developer with a cutting-edge tech stack and a talent for streamlining processes. I pride myself in being a well-rounded developer, able to work on all aspects of app development. My ideal position is working with React, NextJS, Typescript and Tailwind. Pursuing innovative companies where my passion for building cutting-edge applications can shine.

**Comment by jaydeepingle** [comment]
Location: Bellevue, WA
Remote: Open to remote positions
Willing to relocate: Yes
Technologies: React, Java, Python, TypeScript
Resume/CV: https://www.linkedin.com/in/jaydeep-ingle
Website: https://www.github.com/jaydeepingle
Email: inglejaydeep@gmail.com

**Comment by AhmedTakeshy** [comment]
Location: Turkey
Remote: Yes
Willing to relocate: No
Technologies: HTML, CSS, Javascript, ReactJS, NextJS, Redux & redux-toolkit, react-router-dom, Bootstrap, Tailwind, Git & Github, and jQuery(familiar)
Resume/CV: https://www.linkedin.com/in/ahmed-abdelsamie-a1a46a166/
Website: https://takeshy.works
Email: Ahmedtakeshy7@gmail.com

**Comment by Chukwu3meka** [comment]
Location: Nigeria

Remote: Yes

Willing to relocate: Yes

Technologies: Next.js, Typescript, Express, MongoDB, Material-UI, Firebase, oAuth, Mongoose, git, Redux,  APIs, currently learning Zig

Resume/CV: https://chukwuemeka.soccermass.com/assets/Chukwuemeka's%20R%C3%A9sum%C3%A9.pdf

Website: https://github.com/Chukwu3meka

Email: maduekwepedro@gmail.com

LinkedIn: https://www.linkedin.com/in/chukwu3meka

**Comment by zer0and1** [comment]
Location: Fort Mohave, Arizona, US
Remote: Yes
Willing to relocation: No
Technologies: Ember, Next, React, Angular, Vue, Flutter, Lit, Node, Typescript, Javascript, CSS
Resume/CV: https://jeffrey-childers.me
Website: https://jeffrey-childers.me
Linkedin: https://linkedin.com/in/jeffrey-childers-tech
Email: royalmediaus@gmail.com

**Comment by kparf** [comment]
Location: Belarus
Remote: Yes
Willing to relocate: No
Technologies: Next.js, React, Node, Typescript, Prisma
Resume/CV: https://www.linkedin.com/in/kiryl-parfiankou-43a47452
Email: kiryl.parfiankou.dev@gmail.com

**Comment by devir18** [comment]
Hiring for Frontend UI angular Developer at Wroclaw.
Is anybody interested? 

Could you share your information if you are looking for work? 

Location:
Willing to work onsite (once a week in Wroclaw):
Legally authorized to work in Poland (Yes/No): 
Technologies:
Resume/CV:
Website:
Email:

**Reply by codepool867** [reply]
Dear Client,
I am a Senior Web Developer with 8+ years of experience.
When it comes to Angular, I have more than 5 years of experience.
I have worked for several companies in Europe and United States.
I am able to relocate to Wroclaw after 2 months later if things are going well.
Currently located in Belgrade, Serbia.
Yes, I am legally authorized to work in Poland.
Resume:  [YANKO YANICHIEVICH_LATEST.docx](https://github.com/vercel/next.js/files/11407109/YANKO.YANICHIEVICH_LATEST.docx)
Website: https://yanko-portfolio.netlify.app/
Email: serviusapolum@gmail.com

**Comment by omarfrt** [comment]
Location: Morocco
Remote: yes
Willing to relocate: yes
Technologies: Nodejs, expressjs, React, Nextjs, Postgress, NoSql, MongoDB
Resume/CV: [github](https://github.com/omarfrt)
Website: [LinkedIn](https://www.linkedin.com/in/omar-fertat-30953717b/)
Email: [omarfertat96@gmail.com](omarfertat96@gmail.com)

**Comment by devir18** [comment]
Hiring for Fullstack Angular 11 Developer at Wroclaw.
Is anybody interested?

Could you share your information if you are looking for work?

Location:
Willing to work onsite (thrice a week in Wroclaw office):
Legally authorized to work in Poland (Yes/No):
Technologies:
Resume/CV:
Website:
Email:

**Comment by devir18** [comment]
Hiring for Angular 8+ developer at Wroclaw.
Is anybody interested?

Could you share your information if you are looking for work?

Location:
Willing to work onsite (thrice a week in Wroclaw office):
Legally authorized to work in Poland (Yes/No):
Technologies:
Resume/CV:
Website:
Email:
